src/pages/selectSchedule/components/selectPriority/PriorityColumn.tsx

```diff
@@ -11,14 +11,27 @@ function PriorityColumn({ date, timeSlots }: ColumnStructure) {
     ([, slot]) => slot.date === date,
   );
 
+  const changePriorityValue = (priority: number) => {
+    switch (priority) {
+      case 3:
+        return 1;
+      case 2:
+        return 2;
+      case 1:
+        return 3;
+      default:
+        throw Error(`올바르지않은 priority ${priority}`);
+    }
+  };
+
   const getPriorityColumntyle = (selectedEntryId?: number, priority?: number) => {
```

Comment:
여기는 제가 작성한 코드이긴 한데, priority가 undefined인 경우는 없어서 다음과 같이 바꿔주시겠어요? 🙏
그리고 이제보니 함수명이 getPriorityColumnStyle이 되어야하는데 오타가났네요..ㅎㅎ 이것도 같이 바꿔주시면 감사하겠습니다

```suggestion
  const getPriorityColumnStyle = (priority:number, selectedEntryId?: number) => {
```